GPU/GPUTracking/TPCClusterFinder/GPUTPCClusterFinder.cxx

```diff
@@ -90,7 +91,7 @@ void GPUTPCClusterFinder::RegisterMemoryAllocation()
 {
   AllocateAndInitializeLate();
   mRec->RegisterMemoryAllocation(this, &GPUTPCClusterFinder::SetPointersInput, GPUMemoryResource::MEMORY_INPUT | GPUMemoryResource::MEMORY_GPU | GPUMemoryResource::MEMORY_STACK, "TPCClustererInput");
-  mRec->RegisterMemoryAllocation(this, &GPUTPCClusterFinder::SetPointersScratch, GPUMemoryResource::MEMORY_SCRATCH | GPUMemoryResource::MEMORY_STACK, "TPCClustererScratch", GPUMemoryReuse{GPUMemoryReuse::REUSE_1TO1, GPUMemoryReuse::ClustererScratch, (unsigned short)(mISlice % mRec->GetProcessingSettings().nTPCClustererLanes)});
+  mScratchId = mRec->RegisterMemoryAllocation(this, &GPUTPCClusterFinder::SetPointersScratch, GPUMemoryResource::MEMORY_HOST | GPUMemoryResource::MEMORY_GPU | GPUMemoryResource::MEMORY_SCRATCH | GPUMemoryResource::MEMORY_STACK, "TPCClustererScratch", GPUMemoryReuse{GPUMemoryReuse::REUSE_1TO1, GPUMemoryReuse::ClustererScratch, (unsigned short)(mISlice % mRec->GetProcessingSettings().nTPCClustererLanes)});
```

Comment:
Please provide the `GPUMemoryResource::MEMORY_HOST | GPUMemoryResource::MEMORY_GPU` only in case `if (mRec->GetProcessingSettings().runMC)`, otherwise just leave it out (MEMORY_SCRATCH will then default to either GPU or HOST depending on where the data is processed). We should avoid allocating more memory on the host than needed.